TestCases/compliance-level-3/0059-feel-all-function/0059-feel-all-function-test-01.xml

```diff
@@ -37,7 +37,7 @@
 
     <testCase id="004">
         <description>return null if any item is not true or false</description>
-        <resultNode name="decision004" type="decision">
+        <resultNode errorResult="true" name="decision004" type="decision">
```

Comment:
Case:
https://github.com/dmn-tck/tck/blob/9f8ba50870259c971b5980d031fe73224635be09/TestCases/compliance-level-3/0059-feel-all-function/0059-feel-all-function.dmn#L37

DMN Spec:

> list of Boolean items

Concur in the result, but I believe is missing the errorFlag to clarify this is an error case